src/Microsoft.DotNet.Cli.Utils/Constants.cs

```diff
@@ -33,5 +33,7 @@ internal static class Constants
                                                           RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? "osx.10.10-x64" : "ubuntu.14.04-x64";
 
         public static readonly string StaticLibSuffix = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? ".lib" : ".a" ;
+
+        public static readonly string[] KnownCompilers = new string[] { "csc", "vbc", "fsc" };
```

Comment:
This doesn't belong here. Known Compilers is strictly a dotnet-build concept. Let's not let it leak.